source/features/separate-draft-pr-button.tsx

```diff
@@ -13,7 +13,7 @@ function init(): void | false {
 		return false;
 	}
 
-	const buttonBar = select('.timeline-comment > :last-child', previewForm)!;
+	const buttonBar = select('.js-slash-command-surface > :last-child', previewForm)!;
 	const createPrButtonGroup = select('.BtnGroup', buttonBar);
```

Comment:
There's a better class now:
```suggestion
	const createPrButtonGroup = select('.hx_create-pr-button');
```

And then use `createPrButtonGroup.before()` instead of `buttonBar.prepend()`